src/umpire/ResourceManager.cpp

```diff
@@ -283,10 +283,12 @@ void ResourceManager::setDefaultAllocator(Allocator allocator) noexcept
   m_default_allocator = allocator.getAllocationStrategy();
 }
 
+// Using unused attribute to silence warning from rocm compiler
 void ResourceManager::addAlias(const std::string& name, Allocator allocator)
 {
   if (isAllocator(name)) {
     auto ra = getAllocator(name);
+    UMPIRE_USE_VAR(ra);
     UMPIRE_ERROR(runtime_error,
                  umpire::fmt::format("Allocator \"{}\" is already an alias for \"{}\"", name, ra.getName()));
```

Comment:
Perhaps instead of the workaround and comment, this could be changed to something like:
```suggestion
                 umpire::fmt::format("Allocator \"{}\" is already an alias for \"{}\"", name, getAllocator.getName()));
```

Reply:
I can try this!

Reply:
```
/g/g0/belcher6/Umpire/src/umpire/ResourceManager.cpp:290:95: error: reference to non-static member function must be called
                 umpire::fmt::format("Allocator \"{}\" is already an alias for \"{}\"", name, getAllocator.getName()));
```
Looks like it's a `fmt` thing...

Reply:
My suggestion was wrong, try:
```suggestion
                 umpire::fmt::format("Allocator \"{}\" is already an alias for \"{}\"", name, getAllocator(name).getName()));
```